src/test/create_deck.test.ts

```diff
@@ -0,0 +1,27 @@
+
+import CREATE_DECK from '../ast/CREATE_DECK';
+import Tokenizer from '../lib/tokenizer';
+
+test('create deck parse should throw if the input doesnt start with Create Deck', () => {
+  let cd = new CREATE_DECK();
+  Tokenizer.theTokenizer = undefined;
+  Tokenizer.makeTokenizer("Test Create Deck", ["Create Deck"]);
+  const parse = () => { cd.parse() };
+  expect(parse).toThrow(Error);
+});
+
+test("create desk parse should throw if name is missing", () => {
+  let cd = new CREATE_DECK();
+  Tokenizer.theTokenizer = undefined;
+  Tokenizer.makeTokenizer("Create Deck with Tags", ["Create Deck", "with Tags"]);
+  const parse = () => { cd.parse() };
+  expect(parse).toThrow(Error);
+});
+
+test("create desk parse should parse successfully", () => {
```

Comment:
Is this supposed to be passing right now?

Reply:
no since deck and tags parsers were not implemented. It seems like tokenizer isn't working properly too since after strings being tokenized, there are still white spaces

Reply:
there are not many tests I can create rn without deck, tags and style etc.

Reply:
haha ok, that's what I thought, just wondering :)